projects/aca-shared/src/lib/components/page-layout/page-layout.module.ts

```diff
@@ -29,9 +29,10 @@ import { PageLayoutErrorComponent } from './page-layout-error.component';
 import { PageLayoutHeaderComponent } from './page-layout-header.component';
 import { PageLayoutComponent } from './page-layout.component';
 import { CommonModule } from '@angular/common';
+import { CoreModule } from '@alfresco/adf-core';
 
 @NgModule({
-  imports: [CommonModule],
+  imports: [CommonModule, CoreModule.forChild()],
```

Comment:
do you need the whole CoreModule here or maybe some particular modules can work?